chevah_build

```diff
@@ -153,48 +190,42 @@ case $OS in
             else
                 export LDFLAGS="$LDFLAGS -L/usr/lib/mps/64 -R/usr/lib/mps/64"
             fi
+        else
+            # On Solaris 11 we can build the new libraries.
+            EXTRA_LIBRARIES=$EXTRA_LIBRARIES_CFFI
+            PIP_LIBRARIES=$PIP_LIBRARIES_EVERYWHERE
         fi
+        BUILD_CFFI="yes"
```

Comment:
I guess the above line should come inside the `else` clause and before the ending `fi` clause. Consequently, the Solaris 10 builds are broken, eg. https://build.chevah.com/builders/python-package-solaris-10/builds/76/steps/build/logs/stdio